ExRam.Gremlinq.Providers.WebSocket/QueryLoggingVerbosity.cs

```diff
@@ -4,6 +4,7 @@ public enum QueryLoggingVerbosity
     {
         None = 0,
         QueryOnly = 1,
-        QueryAndParameters,
+        QueryAndParameters = 2,
```

Comment:
Maybe it's a good idea to make this enum a flags enum

Reply:
I did wonder about that, but then thought that logging parameters or status attributes without a query may not make sense so wasn't quite sure what this would look like with a single enum. Were you envisaging something like this?

```csharp
[Flags]
enum QueryLoggingVerbosity
{
   None = 0,
   Query = 1,
   QueryParameters = 2,
   QueryStatusAttributes = 4
}
```

Would the following settings then be valid?

```csharp
// Log parameters without query
var queryLoggingVerbosity = QueryLoggingVerbosity.QueryParameters;
```

Reply:
Surely valid although not particularly useful. No one's forced to use it like that though.

Reply:
Good point, I will update as above. Thanks :)